plugin/pb2/resource_pb2.py

```diff
@@ -72,69 +64,44 @@
   ],
   containing_type=None,
   serialized_options=None,
-  serialized_start=152,
-  serialized_end=243,
+  serialized_start=206,
+  serialized_end=297,
 )
-_sym_db.RegisterEnumDescriptor(_RESOURCE_HISTORY)
+_sym_db.RegisterEnumDescriptor(_RESOURCEDESCRIPTOR_HISTORY)
 
 
-_RESOURCE = _descriptor.Descriptor(
-  name='Resource',
-  full_name='google.api.Resource',
+_RESOURCEDESCRIPTOR = _descriptor.Descriptor(
+  name='ResourceDescriptor',
+  full_name='google.api.ResourceDescriptor',
   filename=None,
   file=DESCRIPTOR,
   containing_type=None,
   fields=[
     _descriptor.FieldDescriptor(
-      name='pattern', full_name='google.api.Resource.pattern', index=0,
+      name='type', full_name='google.api.ResourceDescriptor.type', index=0,
```

Comment:
I assume this is the newest version of Resource because we have `type` here, and the projects + organization resource names were removed.

Reply:
Yeah, this looks right.